ntbs-service/Pages/ContactDetails/Index.cshtml

```diff
@@ -0,0 +1,111 @@
+﻿@page
+@using ntbs_service.Helpers
+@model ntbs_service.Pages.ContactDetails.IndexModel
+
+@{
+    Layout = "./_Layout";
+    ViewData["Title"] = $"Case Manager details";
+}
+
+<nhs-width-container container-width="Standard">
+    <h2 class="contact-details-fullname"> @Model.ContactDetails.FullName </h2>
+    <div class="contact-details-edit-container">
+        <h3 class="contact-details-section-heading"> Contact details </h3>
+        <a href="@RouteHelper.GetContactDetailsSubPath(ContactDetailsSubPaths.Edit)"
+            class="contact-details-edit-link govuk-link--no-visited-state">
+            Edit
+        </a>
+    </div>
+    <div class="contact-details-container">
+        <nhs-grid-row>
+            <nhs-grid-column grid-column-width="OneThird">
+                <div class="contact-details-label">               
+                    @Html.DisplayNameFor(x => x.ContactDetails.JobTitle)
+                </div>
+            </nhs-grid-column>
+            <nhs-grid-column grid-column-width="OneThird">
+                <div class="contact-details-text"> @Model.ContactDetails.JobTitle </div>
+            </nhs-grid-column>
+        </nhs-grid-row>
+        <nhs-grid-row>
+            <nhs-grid-column grid-column-width="OneThird">
+                <div class="contact-details-label">
+                    @Html.DisplayNameFor(x => x.ContactDetails.PhoneNumberPrimary)
+                </div>
+            </nhs-grid-column>
+            <nhs-grid-column grid-column-width="OneThird">
+                <div class="contact-details-text"> @Model.ContactDetails.PhoneNumberPrimary </div>
+            </nhs-grid-column>
+        </nhs-grid-row>
+        <nhs-grid-row>
+            <nhs-grid-column grid-column-width="OneThird">
+                <div class="contact-details-label">
+                    @Html.DisplayNameFor(x => x.ContactDetails.PhoneNumberSecondary)
+                </div>
```

Comment:
I think these elements want to be a https://service-manual.nhs.uk/design-system/components/summary-list.